### ISSUE TITLE
[Bug]: Crash dumps in About section can not be shared using TalkBack

### ISSUE THREAD

**Issue by mariansima** [mozilla-mobile/fenix#21307]
### Steps to reproduce

1. Have at least one crash report generated in the app
2. Turn TalkBack (or the available screen reader) on
3. Open the 3-dot menu and open Settings > About Firefox (beta, nightly, etc.) > Crashes
4. Attempt to share or open the links on the crash reports using navigation gestures

### Expected behaviour

The corresponding actions are available to the user as per TalkBack description

### Actual behaviour

TalkBack mentions double tap actions that do no work. The user is unable to share crash reports from the screen using the available navigation gestures

![242019311_1206819396736405_7808049172156245100_n](https://user-images.githubusercontent.com/89511597/133432906-91192b4c-c260-46a8-ac84-72141f815a83.gif)

### Device name

Samsung Galaxy A8

### Android version

Android 9

### Firefox release type

Firefox Beta

### Firefox version

93.0.0-beta.2 2021-09-13

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-17884)

**Comment by SoftVision-LorandJanos** [comment]
This issue is no longer reproducible on the latest Nightly 100.0a1 (2022-03-25) build.
Device used: Google Pixel 4 (Android 12).
Closing the ticket as fixed.

**Comment by SoftVision-LorandJanos** [comment]
I managed to reproduce this issue with a Samsung Galaxy Tab S3 (Android 9) device.
Tested on the latest Nightly 100.0a1 (2022-03-28) build.
Reopening the ticket.

**Comment by kbrosnan** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1802975

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
